tests/bluetooth/ctrl_user_ext/src/main.c

```diff
@@ -0,0 +1,27 @@
+/* main.c - Application main entry point */
+
+/*
+ * Copyright (c) 2015-2016 Intel Corporation
```

Comment:
Oticon 2019

Reply:
I reused parts of the init test, so keeping original copy right seems the right thing to do here